dataland-frontend/src/components/forms/RetrieveCompany.vue

```diff
@@ -20,62 +21,53 @@
         />
 
       </FormKit>
-      <button class="btn btn-md orange darken-3" @click="getCompanyByName(true)">Show all companies</button>
+      <Button @click="getCompanyByName(true)" label="Show all companies" />
       <br>
-      <div class="col m12">
-        <table v-if="response">
-          <caption><h4>Company Search</h4></caption>
-          <thead>
-          <tr>
-            <th v-for="(header, i) in ['Name', 'Headquarter', 'Sector','Market Cap', 'Market Cap Date']" :key="i">
-              {{ header }}
-            </th>
-          </tr>
-          </thead>
-          <tbody>
-          <tr v-for="(dataset, index) in response.data" :key="index">
-            <td v-for="(item, i) in dataset.companyInformation" :key="i">
-              <router-link v-if="i === 'companyName'" :to="'/companies/'+ dataset.companyId ">{{ item }}</router-link>
-              <template v-else>
-                {{ item }}
-              </template>
-            </td>
-          </tr>
-          </tbody>
-        </table>
-        <p v-else-if="response_error">The resource you requested does not exist yet. You can create it:
+      <template v-if="action">
+        <DataTable v-if="response" :value="response.data" responsive-layout="scroll">
+          <Column field="companyInformation.companyName" header="COMPANY" :sortable="true" class="surface-0" >
+          </Column>
+          <Column field="companyInformation.industrialSector" header="SECTOR" :sortable="true" class="surface-0"> </Column>
+          <Column field="companyInformation.marketCap" header="MARKET CAP" :sortable="true" class="surface-0"> </Column>
+          <Column field="companyId" header="" class="surface-0"> <template #body="{data}">
```

Comment:
ich denke diese Columns können automatisch in einer Schleife generiert werden

Reply:
wie besprochen, wir behalten